### ISSUE TITLE
Postinstall script fails on windows

### ISSUE THREAD

**Issue by Hypersycos** [WFCD/warframe-relic-data#240]
The current postinstall script
`npx install-peerdeps @wfcd/eslint-config@latest -S`
in package.json fails on windows with the error
`err spawn einval`

It can be bypassed by installing with scripts disabled and then manually installing the es-lint config. I believe it's something to do with [this security patch](https://nodejs.org/en/blog/vulnerability/april-2024-security-releases-2), but I couldn't figure out how to pass the parameter.

**Comment by TobiTenno** [comment]
yes, it's caused by https://github.com/nathanhleung/install-peerdeps/issues/125

given that our CI is on not-Windows, and this doesn't fail in WSL, i'd recommend using WSL (generally i recommend using it over barebacking windows anyway)

**Comment by TobiTenno** [comment]
should be fine now, btw
